lib/iris/coords.py

```diff
@@ -173,7 +159,8 @@ def copy(self, values=None):
 
         return new_metadata
 
-    def _sanitise_array(self, src, ndmin):
+    @staticmethod
+    def _sanitise_array(src, ndmin):
```

Comment:
We could still make _sanitise_array a static method in #7

Reply:
Yes, if it seems sensible.